src/main/webapp/itinerary-overview.html

```diff
@@ -2,50 +2,62 @@
 <html>
 <head>
     <meta
-        charset="UTF-8"
-        name="viewport"
-        content="width=device-width, initial-scale=1.0"
+    charset="UTF-8"
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
     />
     <title>Itinerary Overview</title>
     <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
     <link
-        href="https://fonts.googleapis.com/icon?family=Material+Icons"
-        rel="stylesheet"
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
I recommend sticking to one editor/style since it looks like the autoformatter changes the formatting of indents and other code blocks a lot for you.

Reply:
Wait sorry I'm not even too sure what changed, what do you recommend I get rid of?

Reply:
If you look at the Files Changed, there's a lot of line diffs due to spacing/indentation. Usually, these shouldn't change every time someone changes a file, so it's best to avoid formatting changes to files that are already existing in master.